test/lib/headers_test.rb

```diff
@@ -428,6 +428,28 @@ def test_request_out_http_referer_subdomain_with_custom_lang_code
       assert_equal('http://wovn.io/test', request_out_env['HTTP_REFERER'])
     end
 
+    def test_request_out_custom_domain
+      settings = Wovnrb.get_settings({
+                                       'url_pattern' => 'custom_domain',
+                                       'custom_domain_langs' => {
+                                         'en' => { 'url' => 'wovn.io' },
+                                         'ja' => { 'url' => 'ja.wovn.io' }
+                                       }
+                                     })
+      store = Wovnrb.get_store(settings)
+      env = Wovnrb.get_env({
+                             'HTTP_REFERER' => 'ja.wovn.io',
+                             'SERVER_NAME' => 'ja.wovn.io',
+                             'REQUEST_URI' => '/dummy'
+                           })
+      url_lang_switcher = UrlLanguageSwitcher.new(store)
+      header = Wovnrb::Headers.new(env, settings, url_lang_switcher)
+
+      request_out_env = header.request_out('ja')
```

Comment:
can you remove this unused parameter. the actual definition as well

Reply:
Removed in c8c339a36ac2904663a1b7df4afae12d714c7f83.